src/api/lib/tasks/dev.rake

```diff
@@ -109,6 +109,12 @@ namespace :dev do
         source_package: ruby_iggy
       )
 
+      create(:bs_request_action_submit_with_diff,
```

Comment:
Instead of extending this method, we could call tasks declared in independent files.
Like we do [here](https://github.com/openSUSE/open-build-service/blob/b4783e908723bc26796767d6972f14f203fb2563/src/api/lib/tasks/dev.rake#L231) to create request with multiple actions.

We can rename the file [lib/tasks/dev/multiple_actions_request.rake](https://github.com/openSUSE/open-build-service/blob/master/src/api/lib/tasks/dev/multiple_actions_request.rake) to `lib/tasks/dev/requests.rake` and move all that code there.